src/MQTTClient.php

```diff
@@ -25,10 +25,15 @@ class MQTTClient implements ClientContract
 {
     use OffersHooks;
 
+    const EXCEPTION_CONNECTION_SOCKET_ERROR        = 1000; // from 1000 to 1255
+    const EXCEPTION_CONNECTION_TLS_ERROR           = 2000; // from ??? to ???
     const EXCEPTION_CONNECTION_FAILED              = 0001;
     const EXCEPTION_CONNECTION_PROTOCOL_VERSION    = 0002;
     const EXCEPTION_CONNECTION_IDENTIFIER_REJECTED = 0003;
     const EXCEPTION_CONNECTION_BROKER_UNAVAILABLE  = 0004;
+    const EXCEPTION_CONNECTION_BAD_LOGIN           = 0005;
+    const EXCEPTION_CONNECTION_NOT_AUTHORIZED      = 0006;
+    const EXCEPTION_CONNECTION_UNKNOWN_ERROR       = 0500;
```

Comment:
Because initially I thought you were mapping 1:1 MQTT error codes, as there are potentially 255 error codes in the protocol i wanted to leave enough space for them. But afterwards I realized they are offsetted by 1 so...

Also, `0099` is not a legal octal value :facepalm:

Reply:
🤦‍♂️